### ISSUE TITLE
Node fails to join as stand-by

### ISSUE THREAD

**Issue by ktsakalozos** [canonical/raft#192]
We have reports in https://github.com/ubuntu/microk8s/issues/2144 where a node fails to start as a stand-by node with the following error:

```
Apr 03 02:26:47 microk8s-hades-node-3 microk8s.daemon-kubelite[17016]: Error: start node: raft_start(): io: load closed segment 0000000000001025-0000000000001024: entries batch 1 starting at byte 8: entries count in preamble is zero
Apr 03 02:26:47 microk8s-hades-node-3 microk8s.daemon-kubelite[17016]: F0403 02:26:47.778967   17016 daemon.go:67] API Server exited start node: raft_start(): io: load closed segment 0000000000001025-0000000000001024: entries batch 1 starting at byte 8: entries count in preamble is zero
```

**Comment by MathieuBordere** [comment]
Similar issue in https://github.com/ubuntu/microk8s/issues/2186#issuecomment-841189860

```
Mai 14 12:38:23 mereet.com microk8s.daemon-kubelite[181888]: Error: start node: raft_start(): io: load closed segment 0000000021384298-0000000021384473: entries batch 176 starting at byte 5799224: entries count in preamble is zero
```

**Comment by pedroleaoc** [comment]
Closing this one for now. Probably not an issue anymore.
